lib/ace/editor.js

```diff
@@ -502,6 +502,29 @@ var Editor = function(renderer, session) {
         this.setOption("fontSize", size);
     };
 
+    this.$showCoordinate = function() {
+        if (this.session.$bracketHighlight) {
+            this.session.removeMarker(this.session.$bracketHighlight);
+            this.session.$bracketHighlight = null;
+        }
+
+        // perform highlight async to not block the browser during navigation
```

Comment:
seems to be copy/paste indeed. Please remove from the copied copy everything which is not related to the added functionality.